### PR TITLE
DEV-1246: update deploy to rsync

### PR DESCRIPTION
**Pull request by carylwyatt**
@aelkiss If I don't have any `INCLUDE` files, it will just include everything that isn't listed in `EXCLUDE`, right? I took out the `$INCLUDE` variable from the rsync command.

### CODE REVIEW

**Comment by aelkiss** [comment]
> @aelkiss If I don't have any `INCLUDE` files, it will just include everything that isn't listed in `EXCLUDE`, right? I took out the `$INCLUDE` variable from the rsync command.

I think in this case we would need to include `$DEPLOY_SRC` in the actual rsync command e.g.

```bash
/usr/bin/rsync "$@" $EXCLUDE $DEPLOY_SRC $DEPLOY_DEST
```

I think this confused @moseshll too because `$INCLUDE` is just the list of files/paths to sync, not specifically using the rsync `--include` flag. Maybe we should change its name elsewhere.

File: bin/rsync.app [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+shift
+
+DEPLOY_DEST=${DEST_PREFIX}www/wordpress-hathitrust
+DEPLOY_SRC=/htapps/test.www/wordpress-hathitrust
+
+EXCLUDE=$(cat <<EOT
+  --exclude .github
+  --exclude .git
+  --exclude .gitignore
+  --exclude $DEPLOY_SRC/node_modules
```
Comment (aelkiss) [inline bin/rsync.app]: I *think* this should just be `--exclude node_modules`

File: bin/rsync.app [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+EOT
+)
+
+/usr/bin/rsync "$@" $EXCLUDE $DEPLOY_DEST
```
Comment (aelkiss) [inline bin/rsync.app]: Need to specify the source somewhere here, e.g. `/usr/bin/rsync "$@" $EXCLUDE $DEPLOY_SRC $DEPLOY_DEST`

**Review by aelkiss — CHANGES_REQUESTED** [review]
See comment -- need to include `$DEPLOY_SRC` in

File: bin/rsync.app (hunk 1 above)
Comment (carylwyatt) [reply bin/rsync.app]: I wasn't sure on that one, thanks!

File: bin/rsync.app (hunk 2 above)
Comment (carylwyatt) [reply bin/rsync.app]: That totally makes sense. I was looking at rsync documentation, but totally missed that I didn't include the src in the command. Thanks!

**Review by aelkiss — APPROVED** [review]
Looks like it should work!

**Pull request merged**